Zend/zend_builtin_functions.stub.php

```diff
@@ -68,18 +68,18 @@ function get_included_files(): array {}
 /** @alias get_included_files */
 function get_required_files(): array {}
 
-function trigger_error(string $message, int $error_type = E_USER_NOTICE): bool {}
+function trigger_error(string $message, int $error_level = E_USER_NOTICE): bool {}
 
 /** @alias trigger_error */
-function user_error(string $message, int $error_type = E_USER_NOTICE): bool {}
+function user_error(string $message, int $error_level = E_USER_NOTICE): bool {}
 
 /** @return string|array|object|null */
-function set_error_handler(?callable $error_handler, int $error_types = E_ALL) {}
+function set_error_handler(?callable $callback, int $error_level_flags = E_ALL) {}
```

Comment:
I'm not 100% sure here, but `$callback` is what we use normally, and I don't think this usage needs any special handling.

On the other hand, I think  just using`$flags` wouldn't be 100% clear, so I went with `$error_level_flags`, but we could have `$error_levels` too.

Reply:
I would use just `$error_level` here. In this case, I think the term is fine both for the single and bitwise case. I would consider `E_ALL` an "error level".

Reply:
Eh, `$error_levels` is probably still better than the singular :)